misc/python/materialize/cli/mzcompose.py

```diff
@@ -37,7 +37,7 @@
 from materialize import ROOT, ci_util, mzbuild, mzcompose, spawn, ui
 from materialize.ui import UIError
 
-MIN_COMPOSE_VERSION = (1, 24, 0)
+MIN_COMPOSE_VERSION = (2, 6, 0)
```

Comment:
This `MIN_COMPOSE_VERSION` is still being compared to the string returned by `docker-compose --version` and not to the one returned by `docker compose --version`

Reply:
Maybe we should instead have a `MIN_DOCKER_VERSION` and enforce that?